test.js

```diff
@@ -13,53 +12,47 @@ describe('should find urls in fixture file', function () {
 		'http://github.com'
 	];
 
-	it('should get unique cleaned-up urls from a string', function () {
-		assert.deepEqual(getUrls(fs.readFileSync('fixture.txt', 'utf8')), expected);
-	});
+	t.same(fn(fs.readFileSync('fixture.txt', 'utf8')), expected);
```

Comment:
Same as all the rest of my comments :). I'll leave it to @sindresorhus to decide if those should be rewritten.